core/commands/mv.py

```diff
@@ -0,0 +1,35 @@
+import functools
+import os
+import sublime
+from sublime_plugin import WindowCommand
+
+from ..git_command import GitCommand
+
+
+class GsMvCurrentFileCommand(WindowCommand, GitCommand):
+
+    """
+    Prompt the user for a new name for the current file.
+    """
+
+    def run(self):
+        if self.file_path:
+            parent, base_name = os.path.split(self.file_path)
+            on_done = functools.partial(self.on_done,
+                self.file_path, parent, base_name)
+            v = self.window.show_input_panel("New Name:", base_name,
+                on_done, None, None)
+            name, ext = os.path.splitext(base_name)
+            v.sel().clear()
+            v.sel().add(sublime.Region(0, len(name)))
+
+    def on_done(self, file_path, parent, base_name, new_name):
+        if new_name == base_name:
+            return
+        new_path = os.path.join(parent, new_name)
+
+        self.git("mv", file_path, new_path)
+        v = self.window.find_open_file(file_path)
```

Comment:
I did not see any documentation for `find_open_file` at https://www.sublimetext.com/docs/api-reference, where did you find this method?
How does it handle if you have multiple instances of the same file open?

Reply:
The Sublime 3 docs are now at https://www.sublimetext.com/docs/3/api_reference.html

Reply:
If there are multiple views to the same file, it will just return one of the views.  That's fine, since the retargeting will affect the underlying buffer and all views will be updated.

Reply:
Durp about the docs, I have seen it before.